compiler/rustc_mir_build/src/thir/cx/expr.rs

```diff
@@ -939,9 +939,10 @@ impl<'tcx> Cx<'tcx> {
                 }
             }
 
-            // We encode uses of statics as a `*&STATIC` where the `&STATIC` part is
-            // a constant reference (or constant raw pointer for `static mut`) in MIR
+            // Writing a path to a static in Rust source creates a place expression AKA lvalue
+            // THIR calls it ExprKind::Deref, so here we gin up something to deref
```

Comment:
```suggestion
            // The surface Rust term `STATIC` is compiled to a THIR place expression
            // that looks like `*ADDR_OF_STATIC`, where `ADDR_OF_STATIC` is a
            // reference for safe `static` and a raw pointer for `extern static` and `static mut`.
```

Reply:
this proposed line 944 would be redundant with the extant line 945?